qcodes/dataset/sqlite/queries.py

```diff
@@ -318,7 +318,7 @@ def _expand_data_to_arrays(data: List[List[Any]], paramspecs: Sequence[ParamSpec
             for element in numeric_elms:
                 row[element] = np.full_like(row[first_array_element],
                                             row[element],
-                                            dtype=np.float)
+                                            dtype=np.double)
```

Comment:
Probably cleaner `np.dtype(np.float64)`

Reply:
because of the explicit "64"?

Reply:
analogiues to complex128 below, i'd suport float64 here.

Reply:
Yes, Double is always equivalent to 64 bit as far as I know.